src/blockstore.js

```diff
@@ -45,6 +45,7 @@ function createBaseStore (store) {
       return store.open()
     },
 
+    // @ts-ignore TODO: how to not have ts freak out about this
```

Comment:
What is the issue here?

Reply:
It can't work out that we are yielding only CIDs or Blocks and not a mixture of both so we are, in fact, compliant with the interface.

Maybe it could use a better comment 😆

Reply:
I've added a better comment